### PR TITLE
plugins more readable

### PR DESCRIPTION
**Pull request by pedromorgan**
some editng to make more manual style

### CODE REVIEW

File: docs/plugins/plugins.md [hunk 1]
```diff
@@ -8,56 +8,48 @@ previous_url: /plugins
...
+NativeScript plugins are [npm packages](https://docs.npmjs.com/getting-started/packages) with added functionality. Finding, installing, and removing NativeScript plugins works the same way as eg: a Node.js server or front-end web development.
 
-* [Finding Plugins](#finding-plugins)
+* [Existing Community Plugins](#existing-community-plugins)
 * [Installing Plugins](#installing-plugins)
 * [Removing Plugins](#removing-plugins)
 
-> **Tip:** Alternatively, you can search, install, and remove plugins with [NativeScript Sidekick](https://www.nativescript.org/nativescript-sidekick). Sidekick also offers starter kits, single page templates, cloud-based builds for iOS and Android, and app store publishing.
+> **Top Tip:** Use [NativeScript Sidekick](https://www.nativescript.org/nativescript-sidekick) to search, install, and remove plugins with. Sidekick also has starter kits, single page templates, cloud-based builds for iOS and Android, and app store publishing with more in development.
```
Comment (NickIliev) [inline docs/plugins/plugins.md:17]: Perhaps we should stick to a global naming convention and use exclusively **Tip** or **Note**

File: docs/plugins/plugins.md [hunk 2]
```diff
@@ -8,56 +8,48 @@ previous_url: /plugins
...
 
-Once you’ve found the plugin you need, install the plugin into your app using the `tns plugin add` command.
-
-```
-tns plugin add <plugin-name>
-```
+To install a plugin dependancy into an app use the `tns plugin add` command.
 
-For example, the following command installs the [NativeScript camera plugin](http://market.nativescript.org/plugins/nativescript-camera).
+Eg: the following command installs the [NativeScript camera plugin](http://market.nativescript.org/plugins/nativescript-camera) plugin with a few. Note it also ammends the package.json file and a few other `tns` tricks.
```
Comment (NickIliev) [inline docs/plugins/plugins.md:32]: Again as a reading convention, it would be clearer to use "For example" instead of e.g.

**Comment by etabakov** [comment]
@pedromorgan Would you mind to address the feedback from @NickIliev, so we can merge this PR?

**Comment by etabakov** [comment]
Closing the PR due to inactivity.

**Pull request closed without merging**